init.ps1

```diff
@@ -57,7 +61,17 @@ function AppendContent($Message, $Uri, $BuildScriptPath, [switch]$Enable, [hasht
     if($enable){
         Write $Message
         $progressPreference = 'silentlyContinue'
-        $content = (Invoke-WebRequest -Uri $Uri -UseBasicParsing -Headers @{"Cache-Control"="no-cache"} ).content
+		$requestResult = (Invoke-WebRequest -Uri $Uri -UseBasicParsing -Headers @{"Cache-Control"="no-cache"} )

```

Comment:
For local dev / verification just create temp directory next to the repo and use `..\script\init.ps1`